packages/plugin-compiler/src/main/kotlin/io/realm/kotlin/compiler/RealmModelSyntheticPropertiesGeneration.kt

```diff
@@ -560,6 +591,8 @@ class RealmModelSyntheticPropertiesGeneration(private val pluginContext: IrPlugi
                                     putValueArgument(
                                         arg++,
                                         if (type == linkingObjectType) {
+                                            // FIXME Link property name must be the persisted name of the backlink (linking object)
```

Comment:
Added it more as a reminder for this PR (along with the TODO item in the PR description) since it needs to be fixed together with these changes, otherwise it will break backlinks if adding `@PersistedName` annotation to a backlink. So in this case, a separate issue shouldn't be created right?